lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -12,6 +12,13 @@ defmodule ExUnit.Case do
       It should be enabled only if tests do not change any global state.
       Defaults to `false`.
 
+    * `:async_partition_key` - configures async tests in this module to run
```

Comment:
ExUnit already has the concept of partitions so, to avoid confusion, I would prefer to call this `async_group`.

Reply:
Good point, I'll change it to group 👍

Reply:
I've changed the option name in https://github.com/elixir-lang/elixir/pull/13897/commits/ade76058e69def1443c74de447c9c9bee659608b.